protos-frontend/frontend/FrontendService.proto

```diff
@@ -59,4 +64,32 @@ message TriggerEmailVerificationRequest {
 }
 message TriggerEmailVerificationResponse {
     enums.VerificationResponseType response_type = 1;
+}
+
+//Pan verification
+message GetPanVerificationScreenRequest{
+
+}
+
+
+message GetPanVerificationScreenResponse {
+    enums.VerificationResponseType response_type = 1;
```

Comment:
I think we should not use the verification response type here. If you see inside that enum we have something as otp sent and all which flow specific. Hence we should not have this value.